src/modules/apps.js

```diff
@@ -7,40 +109,150 @@ export default {
     description: 'List your installed VTEX apps',
     handler: () => {
       log.debug('Starting to list apps')
-      log.info('You have no installed apps')
+      workspaceAppsClient().listDependencies(
+        getAccount(),
+        getDevWorkspace(getLogin()),
+        'storefront'
```

Comment:
@brenoc Yeah, list only for storefront doesn't make sense.
BUT, a service is required. Should we make the service an argument?